mxcubeweb/core/components/lims.py

```diff
@@ -323,3 +323,34 @@ def synch_with_lims(self, lims_name):
                 self.sample_list_sync_sample(sample_info)
 
         return self.sample_list_get()
+
+    def get_lims_samples(self, lims_name: str) -> dict:
+        """
+        Gets samples from LIMS and filters to include only LIMS-linked entries.
+
+        This method synchronizes the sample list with the specified LIMS,
+        filters for entries that have a `limsID`, and sets the new sample list
+        accordingly.
+
+        Args:
+            lims_name (str): Name of the LIMS system to synchronize with.
```

Comment:
```suggestion
            lims_name: Name of the LIMS system to synchronize with.
```

Skip the type in the docstring if there is already a type hint annotation. Sphinx documentation generator and all the other tools will know to do the right thing.